packages/react-native-client/src/stats/useRTCStatistics.ts

```diff
@@ -14,7 +14,7 @@ export function useRTCStatistics(refreshInterval: number) {
   // Calculates diff between pervious and current stats,
   // providing end users with a per second metric.
   const processIncomingStats = useCallback(
-    (statistics: RTCStats[], stats: RTCStats) => {
+    (statistics: ReadonlyArray<RTCStats>, stats: Readonly<RTCStats>) => {
```

Comment:
What is the benefit of changing of making both arguments readonly?

Reply:
It is just better :)
- by marking params as readonly, you are guarantee that variable won't be updated inside of the function (for example, you won't be able to call `statistics.sort()` as this modify passed array, and would be unexpected side effect)
- if someone would like to use this function, and uses readonly arrays/objects (as good practice), adding it as part of declaration will make developers life easier.

Reply:
if some would want to manipulate on the data he will just get rid of Readonly or work on its mutable copy. Although i do not fully support this change it can stay.

Reply:
>if some would want to manipulate on the data he will just get rid of Readonly or work on its mutable copy. Although i do not fully support this change it can stay.

We can discuss it offline. But my experience is that code where all methods accept only readonly arrays is just better and less error prone